src/providers/vk.ts

```diff
@@ -0,0 +1,54 @@
+import type { OAuthConfig, OAuthUserConfig } from "."
+
+export interface VkProfile {
+  response: Array<{
+    id: number
+    first_name: string
+    last_name: string
+    photo_100: string
+    can_access_closed: boolean
+    is_closed: boolean
+  }>
+}
+
+export default function VK<
+  P extends Record<string, any> = VkProfile
+>(options: OAuthUserConfig<P>): OAuthConfig<P> {
+  const apiVersion = "5.126" // https://vk.com/dev/versions
```

Comment:
```suggestion
  const apiVersion = "5.131" // https://vk.com/dev/versions
```

I tested this, let's update the version.